src/core/core.js

```diff
@@ -471,6 +471,7 @@ var p5 = function(sketch, node, sync) {
   // assume "global" mode and make everything global (i.e. on the window)
   if (!sketch) {
     this._isGlobal = true;
+    p5.instance = this;
```

Comment:
If we want to be really friendly, we could actually add a check here to see if `p5.instance` is non-`null`, and log a friendly warning if so.  Hopefully that should never happen, but it could potentially cause some hard-to-catch bugs if it ever does.